src/lib/holocene/input/chip-input.svelte

```diff
@@ -10,13 +12,39 @@
   export let required = false;
   export let hintText = '';
   export let validator: (value: string) => boolean = () => true;
+  const values = writable<string[]>(chips);
```

Comment:
What's the motivation behind making this a store?

Reply:
I wanted some way to compare the previous values to the new values to determine if `shouldScrollToInput` after the input updates. Open to ideas if there's a better way to go about this!